prow/cluster/simple_pipeline_deployment.yaml

```diff
@@ -0,0 +1,33 @@
+kind: Deployment
```

Comment:
There is already a deployment in `pipeline_deployment.yaml`. I am wondering wha's the reason to not reuse it.

Reply:
That flag is for running the jobs in a different cluster.

I think it is too misleading to include these deployment files in the `prow/cluster` directory. With the exception of `starter.yaml` (which we should probably move) all of the files under `prow/cluster` are deployed to the prow.k8s.io instance, but the files you've added wouldn't work for our instance.

It looks like this is a more detailed version of `prow/cmd/pipeline/dev.yaml`. Consider updating that file or adding this one to that directory.

Reply:
I see, it can be actually pointed to the same cluster. It's true there is an additional configuration step to set up that secret. 

Maybe it shouldn't fail with an error if the provided file is empty/missing. Sounds like a warnings message and use the same cluster is more appropriate in that case.

Reply:
> It looks like this is a more detailed version of prow/cmd/pipeline/dev.yaml. Consider updating that file or adding this one to that directory.

Ah I didn't even see dev.yaml! For now I've moved these examples into `cmd/pipeline` and `cmd/crier`